### PR TITLE
NAS-110793 / 12.0 / remove lxml and xml.etree from identify_freebsd

### PR DESCRIPTION
**Pull request by yocalebo**
- remove `lxml` and `xml` from `identify_freebsd` and use the built-in `geom.class_by_name.xml` attribute
- while here, remove unused `identify_linux` and `device_info_linux` modules (1 of which actually imports `lxml`)
- if a `{serial}` was requested to be identified, we were calling `geom.class_by_name.xml` twice for which I've made it so that we only call it once

### CODE REVIEW

File: src/middlewared/middlewared/plugins/disk_/identify_freebsd.py
Comment (themylogin) [inline src/middlewared/middlewared/plugins/disk_/identify_freebsd.py:74]: Are we sure that XML document values are always whitespace-normalized?

Comment (yocalebo) [reply src/middlewared/middlewared/plugins/disk_/identify_freebsd.py:85]: It's probably not, so I've fixed the logic.

**Comment by rick-mesta** [comment]
@themylogin / @yocalebo : What's the status on this one, gents ? Are all issues resolved ?

**Comment by yocalebo** [comment]
> @themylogin / @yocalebo : What's the status on this one, gents ? Are all issues resolved ?

I've addressed the review, just waiting on approval/re-review from @themylogin

**Review by themylogin — APPROVED** [review]

**Comment by rick-mesta** [comment]
Noice ! Thx @themylogin 😁

**Pull request merged**
